test/integration/api.bats

```diff
@@ -880,12 +945,43 @@ function teardown() {
 	[[ "${lines[1]}" != *"Paused"* ]]
 }
 
-# FIXME
 @test "docker version" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	# version
+	run docker_swarm version
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -ge 8 ]
+
+	# verify
+	client_reg='^Client version: [0-9]+\.[0-9]+\.[0-9]+$'
```

Comment:
This is failing on Jenkins.

I believe because we use docker master, and the version is `1.7.0-dev`.